app/views/articles/new.html.haml

```diff
@@ -0,0 +1,11 @@
+= form_with model: Article.new, url: articles_path, local: true do |form|
```

Comment:
Please see my comment on the `ArticlesController`. Also, when you use the `model:` setting, you can probably omit the `url: `

Reply:
+1 for remembering `local: true`